gcp_variant_transforms/beam_io/vcfio_test.py

```diff
@@ -72,7 +72,7 @@
     '19	12	.	C	<SYMBOLIC>	49	q10	AF=0.5	GT:PS:GQ	0|1:1:45	.:.:.\n')
 GVCF_LINE = '19	1234	.	C	<NON_REF>	50	.	END=1236	GT:GQ	0/0:99\n'
 
-def _get_sample_variant_1(file_name=''):
+def _get_sample_variant_1(file_name='', use_1_based_coordinate=False):
```

Comment:
Please add the same logic to `_get_sample_variant_2` and 3.

Also if you could repeat `test_file_pattern_verify_details` with 1_based flags then I feel we have enough unit tests to cover this new execution branch.